### ISSUE TITLE
Failed to build OKD openshift-installer with libvirt support (fcos branch)

### ISSUE THREAD

**Issue by rarguello** [openshift/installer#3412]
# Platform:
Building OKD openshift-installer with libvirt support (fcos branch) on Fedora 31 (x86_64).

# What happened?
Trying to build OKD openshift-installer with libvirt support gives an error:

```console
$ TAGS=libvirt hack/build.sh
   <snip>
go: github.com/LorbusChris/machine-config-operator@v0.0.1-0.20200323222512-db1a46cb4bbb: invalid version: unknown revision db1a46cb4bbb
```

# What you expected to happen?
openshift-installer with libvirt support builds correctly.

# How to reproduce it (as minimally and precisely as possible)?
Clone the openshift-installer repo and checkout the "fcos" brach:

```console
$ mkdir -p ~/go/src/github.com/openshift/
$ cd ~/go/src/github.com/openshift/
$ git clone https://github.com/openshift/installer.git
$ cd installer
$ git checkout fcos
```

Build with libvirt support:

```console
$ TAGS=libvirt hack/build.sh
+ minimum_go_version=1.13
++ go version
++ cut -d ' ' -f 3
+ current_go_version=go1.13.6
++ version 1.13.6
++ IFS=.
++ printf '%03d%03d%03d\n' 1 13 6
++ unset IFS
++ version 1.13
++ IFS=.
++ printf '%03d%03d%03d\n' 1 13
++ unset IFS
+ '[' 001013006 -lt 001013000 ']'
+ MODE=release
++ git rev-parse --verify 'HEAD^{commit}'
+ GIT_COMMIT=ade603e8d9a0f978a1b5932dd3069e2bb01eb92a
++ git describe --always --abbrev=40 --dirty
+ GIT_TAG=unreleased-master-2475-gade603e8d9a0f978a1b5932dd3069e2bb01eb92a
+ GOFLAGS=-mod=vendor
+ LDFLAGS=' -X github.com/openshift/installer/pkg/version.Raw=unreleased-master-2475-gade603e8d9a0f978a1b5932dd3069e2bb01eb92a -X github.com/openshift/installer/pkg/version.Commit=ade603e8d9a0f978a1b5932dd3069e2bb01eb92a'
+ TAGS=libvirt
+ OUTPUT=bin/openshift-install
+ export CGO_ENABLED=0
+ CGO_ENABLED=0
+ case "${MODE}" in
+ LDFLAGS=' -X github.com/openshift/installer/pkg/version.Raw=unreleased-master-2475-gade603e8d9a0f978a1b5932dd3069e2bb01eb92a -X github.com/openshift/installer/pkg/version.Commit=ade603e8d9a0f978a1b5932dd3069e2bb01eb92a -s -w'
+ TAGS='libvirt release'
+ test '' '!=' y
+ go generate ./data
go: github.com/LorbusChris/machine-config-operator@v0.0.1-0.20200323222512-db1a46cb4bbb: invalid version: unknown revision db1a46cb4bbb
```

**Comment by vrutkovs** [comment]
Can't reproduce, https://github.com/LorbusChris/machine-config-operator/commit/db1a46cb4bbb is a valid commit and the installer builds as expected here

**Comment by rarguello** [comment]
I can't find that commit!

```
$ git clone https://github.com/LorbusChris/machine-config-operator.git
Cloning into 'machine-config-operator'...
remote: Enumerating objects: 32733, done.
remote: Total 32733 (delta 0), reused 0 (delta 0), pack-reused 32733
Receiving objects: 100% (32733/32733), 20.47 MiB | 8.09 MiB/s, done.
Resolving deltas: 100% (17085/17085), done.
$ cd machine-config-operator
$ git checkout db1a46cb4bbbd9b406a81c0dad446f9e9aa41bfc
fatal: reference is not a tree: db1a46cb4bbbd9b406a81c0dad446f9e9aa41bfc
```

**Comment by rarguello** [comment]
Fixed by commit a29835acda923d423ca26caa9ad68ec38ef4b2ab.

Building OKD openshift-installer with libvirt support works now.

```
$ TAGS=libvirt hack/build.sh
+ minimum_go_version=1.13
++ go version
++ cut -d ' ' -f 3
+ current_go_version=go1.13.9
++ version 1.13.9

...

+ go build -mod=vendor -ldflags ' -X github.com/openshift/installer/pkg/version.Raw=unreleased-master-2487-ge0b9dedd751543fbc01066a3049ff000e60b1459 -X github.com/openshift/installer/pkg/version.Commit=e0b9dedd751543fbc01066a3049ff000e60b1459 -s -w' -tags 'libvirt release' -o bin/openshift-install ./cmd/openshift-install
$
```
